skills/ramenctl-test-clean/SKILL.md

```
@@ -0,0 +1,65 @@
+```
+
+Use `--config <file>` if the config file is not the default `config.yaml`.
+
+The command unprotects and undeploys test applications, then cleans the
+test environment (channels, namespaces).
+
+### Step 2: Check the result
+
+**On success:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)